pipeline/terraform/env.sh.example

```diff
@@ -35,4 +35,37 @@ export TF_VAR_ecs_asg_min_size=2
 export TF_VAR_ecs_asg_max_size=2
 
 # HTTPS certificate ARN from AWS certificate manager
-export TF_VAR_certificate_arn=arn:aws:acm:us-east-1:123412341234:certificate/12341234-3144-4c26-ba3c-123412341234
\ No newline at end of file
+export TF_VAR_certificate_arn=arn:aws:acm:us-east-1:123412341234:certificate/12341234-3144-4c26-ba3c-123412341234
+
+#schema name for the rds-mysql database
+export TF_VAR_identifier=demodb
+
+#database engine specifying "mysql" as the database server of choice in RDS
+export TF_VAR_engine=mysql
+
+#Version of mysql server to launch in RDS
+export TF_VAR_engine_version=5.7.19
```

Comment:
I see mysql has a latest version of v5.7.23.  Why is this set to v5.7.19?

Reply:
There is no RDS support for mysql v5.7.23. However the latest version supported is v5.7.21. I haven't tested the v5.7.21 but thought not to use the latest supported version just in case there are any unforeseen bug. TF currently uses v5.7.19 in the default version.

Reply:
My vote would be to go with the latest supported version of MySQL. (It's highly likely that the TF docs are just slightly out of date. Version references in docs hardly ever keep up properly with "latest.")

Reply:
Agree.  Upgrade to the latest and greatest.  Its a minor version change, so any bugs should be minor in terms of the using the version.  We can verify usage during practice sessions.  You might make a small comment mentioning the reason for using the current version too.

Reply:
Updated script to latest support version - v5.7.21